dev/com.ibm.ws.jpa.container/src/com/ibm/ws/jpa/container/osgi/internal/JPAComponentImpl.java

```diff
@@ -1155,4 +1155,5 @@ public JPAVersion getJPAVersion() {
             return JPAVersion.UNKNOWN;
         }
     }
+
```

Comment:
This file had no real changes; I would recommend removing from your PR

Reply:
done